### DISCUSSION TITLE
What does nprobe do when the index is FLAT?

### DISCUSSION THREAD

**Discussion by JeroenVanOort** [milvus-io/milvus#3917] [Q&A and General discussion] [answered]
I think I understand the significance of `nprobe` when using any index but FLAT. But does it's value make any difference when the index is FLAT?

**Comment by sahuang** [comment, marked as answer]
`nprobe` will never be checked in `FLAT` index. In fact we do not care about the parameter when using `FLAT`.

**Reply by JeroenVanOort** [reply]
Thanks!
